repose-aggregator/tests/release-verification/src/config/scripting.cfg.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scripting xmlns="http://docs.openrepose.org/repose/scripting/v1.0"
+           language="groovy">
+    if (request.getHeader('X-Auth-Token') == null) {
+        request.addHeader('X-Auth-Token', '12345678901234567890')
+    }
+</scripting>
```

Comment:
It should probably call through the filter chain, as @Mario-Lopez pointed out. I'm actually not sure why it's working the way it is. It shouldn't be making it to the origin service...